src/pr_closed.ts

```diff
@@ -1,20 +1,20 @@
-import {TerraformCloudApi} from "./tfc_api";
 import {TerraformCli} from "./tfc_cli";
 import {GithubHelper} from "./gh_helper";
+import { TerraformBackend } from "./types";
 
 export async function handlePrClosed(
-    tfcApi: TerraformCloudApi,
+    tfcApi: TerraformBackend,
     tfcCli: TerraformCli,
     ghHelper: GithubHelper
 ){
     try {
         let outputInit = tfcCli.tfInit();
         console.log(outputInit);
-        let outputDestroy = tfcCli.tfDestroy();
-        console.log(outputDestroy);
     } catch (e: any) {
         console.log('Workspace may not have been initialized', e);
     }
+    let outputDestroy = tfcCli.tfDestroy();
```

Comment:
having this inside the try was swallowing exceptions when the destroy command failed. moved it out so that the tf cli error will bubble up and prevent workspace data from being deleted.